libs/datasets/dataset_utils.py

```diff
@@ -364,3 +365,34 @@ def fill_fields_with_data_source(
     data = pd.concat([existing_df.reset_index(), missing_new_data[columns_to_fill].reset_index(),])
 
     return data
+
+
+def make_binary_array(
+    data: pd.DataFrame,
+    aggregation_level: Optional[AggregationLevel],
+    country,
+    fips,
+    state,
+    states,
```

Comment:
Should more of these have defaults as none?

Reply:
I was considering removing `=None` from all of them since I'm not expecting this function to be called directly :-P
Added =None` to all.